java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1246,11 +1246,9 @@ public Errata publishAsOriginal(User loggedInUser, String advisoryName, List<Str
             catch (ClassCastException e) {
                 // just catch, do not do anything
             }
-            finally {
-                if (cc == null || !cc.isCloned()) {
-                    throw new InvalidChannelException("Cloned channel " +
-                            "expected: " + c.getLabel());
-                }
+            if (cc == null || !cc.isCloned()) {
```

Comment:
That won't be executed in case of `ClassCastException`. I wonder how likely this is to happen though

Reply:
oh right! short nights make bad reviews.